scripts/import_large_tsv/import_large_tsv_python3.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Upload a local tsv to a Terra workspace data model when it is too large to import from Terra UI."""
+import argparse
+from firecloud import api as fapi
+from tqdm import tqdm
+
+
+def upload_tsv_to_workspace(tsv, project, workspace):
+    """Split large TSV file and upload individual smaller TSV files to Terra workspace."""
+    tsv_strings = []
+
+    # open input tsv file
+    with open(tsv, "r") as tsvfile:
+        headers = tsvfile.readline()
+        tsv_data = headers
+
+        # for each line in tsv line, add to header. chunk by 5000 lines.
+        for i, line in enumerate(tsvfile):
+            tsv_data += line
+
+            if i > 0 and i % 5000 == 0:
```

Comment:
can you add a comment to explain this logic? it's not transparent to me how this is working

Reply:
Can you see if these comments make more sense?

Reply:
so much better, nice work!